shell/platform/darwin/common/framework/Headers/FlutterBaseDartProject.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERBASEDARTPROJECT_H_
+#define FLUTTER_FLUTTERBASEDARTPROJECT_H_
+
+#import <Foundation/Foundation.h>
+
+#import "FlutterMacros.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * A set of Flutter and Dart assets used by a `FlutterEngine` to initialize execution.
+ *
+ * TODO(stuartmorgan): Align API with iOS DartProject, and combine.
+ */
+@interface FlutterBaseDartProject : NSObject
```

Comment:
It looks like right now the macOS subclass _only_ adds mirrors. Not sure how much the additional functionality the iOS project definition currently has, but if nothing/not much, we could remove `enableMirrors` and kill the subclass altogether, then just leave this as `FlutterDartProject`, which I think is the desired end-state anyway.

Otherwise, I think we'll be stuck with an empty subclass for each platform forever.